backend/src/seeders/seed.data.ts

```diff
@@ -5,11 +5,11 @@ import { validRelationships } from '../users/user-relationship/entities/user-rel
 import * as bcrypt from 'bcrypt';
 import { CreateChatUserPermissionDto } from '../chats/chat-user-permissions/dto/create-chat-user-permission.dto';
 import { CreateUserAchievementDto } from '../users/user-achievements/dto/create-user-achievement.dto';
-import { CreateMessageDto } from '../chats/message/dto/create-message.dto';
+import { permissionsEnum } from '../chats/chat-user-permissions/entities/chat-user-permission.entity';
 
 class seedUser {
 	name: string;
-	password: string;
+	password?: string;
 	is_intra: boolean;
 	avatar?: string;
 	created_at?: string;
```

Comment:
isnt the created at always something?

Reply:
probably? But doesn't have to be. And it's for seeding anyway. So if we ever want to add a new seeded user, we don't _have_ to set the create date